test/controllers/lab_results_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class LabResultsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Should there be tests in here? You can probably just model after the tests for the immunization controller, and it might get you closer to passing CodeCov.

Reply:
resolved